File: code/go/site/static/code_snippets/getting_started/multiple_events.php
<?php
use starfederation\datastar\ServerSentEventGenerator;

$sse = new ServerSentEventGenerator();
$sse->mergeFragments('<div id="question">...</div>');
$sse->mergeFragments('<div id="instructions">...</div>');
$sse->mergeSignals(['answer' => '...']);
$sse->mergeSignals(['prize' => '...']);

File: code/go/site/static/code_snippets/getting_started/multiple_events.go
import (
	datastar "github.com/starfederation/datastar/code/go/sdk"
)

sse := datastar.NewSSE(w,r)
sse.MergeFragments(`<div id="question">...</div>`)
sse.MergeFragments(`<div id="instructions">...</div>`)
sse.MergeSignals([]byte(`{answer: '...'}`)
sse.MergeSignals([]byte(`{prize: '...'}`)
